assets/js/practice.mjs

```diff
@@ -106,7 +89,8 @@ const startTimer = (time) => {
 
 // start the typing game
 startBtn.addEventListener("click", () => {
-    const quote = makequote();
+    const quote = makequote()
+    
```

Comment:
At line 141 change the last calculation `(extracted_words.length - 2)` to `(extracted_words.length - 1)`